src/components/CreateContract/CreateContractForms/team/container.js

```diff
@@ -57,54 +40,66 @@ export default Wrapped =>
       return this.props.onSuccess(data);
     };
 
-    handleTeamSearch = selectedClientId => {
-
-      getTeamsFromClient(selectedClientId)
-        .then(response => {
-          const teams = response.data;
-          const teamsFormatted = teams.reduce((acc, team) => {
-            acc.push({
-              value: team.teamId,
-              displayValue: team.teamName,
-            });
-            return acc;
-          }, []);
-          teamsFormatted.unshift({
-            value: '-1',
-            displayValue: 'Please select a team',
-          });
-          this.setState({
-            teams: teamsFormatted,
-            teamResults: `${teams.length} ${teams.length > 1 ? 'teams' : 'team'} found.`,
-          });
-        })
-        .catch(() => {
-          this.setState({
-            teamResults: 'No Teams Found',
-          });
-        });
+    handleTeamSearch = clientId => {
+      this.props.updateSelectedClientId(clientId);
     };
 
     handleTeamFormReset = resetForm => {
       event.preventDefault();
+      const { updateSelectedTeamId, updateSelectedClientId } = this.props;
       this.setState({
-        teams: [],
         teamResults: '',
       }, () => {
+        updateSelectedTeamId(-1);
+        updateSelectedClientId(-1);
         resetForm();
       });
     };
 
     render() {
+      const { 
+        teamOptions, 
+        clientOptions,
+      } = this.props;
       return (
         <Wrapped
-          teams={this.state.teams}
+          teamOptions={teamOptions}
           teamResults={this.state.teamResults}
-          clients={this.state.clients}
+          clientOptions={clientOptions}
           handleTeamSectionSubmit={this.handleTeamSectionSubmit}
           searchTeam={this.handleTeamSearch}
           handleFormReset={this.handleTeamFormReset}
         />
       );
     }
   };
+
+const mapStateToProps = state => {
+
+
```

Comment:
Purely aesthetic, but this whitespace is a bit jarring. 

Have these files been linted?

Reply:
Space has been removed. No linting errors are flagging for this?

Reply:
I'd suggest adding some for this https://eslint.org/docs/rules/no-multiple-empty-lines { "max": 2 } option seems the best but have a look around.

Reply:
This might be better added as a separate ticket